README.md

```diff
@@ -8,11 +8,32 @@ Notably, the multi-sig process on Flow is associated with two principal accounts
 
 At present, representatives from Blocto, Ichi, FindLabs, Flow Foundation, and an individual member (`@bluesign`) collectively hold multi-sig authority over the service account. Each signer is allocated a weight of 250 units, and to validate a transaction, a cumulative total of 1000 units is necessary. Therefore, a minimum of four signers must jointly provide their signatures to authorize the transaction. The transactions deliberated by these signatories may be related to setting transaction fees, managing/ slashing rewards, updating network-level contracts, adding new nodes, driving height coordinated and other node software upgrades, etc; see [account page](https://developers.flow.com/build/basics/accounts#service-accounts) for more details on the powers and abilities of the Flow service account.
 
-**Role of a multi-signer**
+## Role of a multi-signer
 
 Multi-signers on Flow have a direct hand in shaping Flow’s future. They represent the community, oversee protocol changes, and ensure decisions align with the interests of all Flow participants. They regularly join meetings, some scheduled and others on short notice. On average and under normal circumstances, signers are expected to attend 1-2 meetings per month, each lasting about 10-15 minutes. During busier periods, such as major protocol updates, there may be more frequent and short-notice meetings.
 
-**How to nominate?**
+## 🛠️ Service Committee
+The group of multi-signers are effectively referred to as the service committee.
+
+## 📋 Policy for Multi-signers
+An efficient and smooth multi-sign process is essential to ensure upgrades on Flow are rolled out in a timely manner. Reviewing, approving, and signing transactions requires a time commitment from each member of the service committee.
+
+Equally important is preserving the decentralized nature of the multi-sign process. The service committee must reflect the diversity and independence of the Flow ecosystem and avoid any form of centralization of signing power.
+
+To that end, the following policy applies:
+
+1. Monthly Participation
+Each signer is expected to attend at least one multi-signing meeting per month.
```

Comment:
I think we can make it like missing N signing meetings. I guess service account will be needed less and less as protocol evolves ( probably still a long way )

Reply:
Changed the policy to be more in terms of consecutive signing instead of month.